di_website/templates/contactus/contact_page.html

```diff
@@ -0,0 +1,41 @@
+
+{% extends "layouts/default.html" %}
+
+{% load wagtailcore_tags %}
+
+
+{% block hero %}
+    {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+<section class="section">
+    <div class="row row--narrow">
+        <div class="max-meter">
+             <div class="is-typeset">
+                {% if page.intro %}
+                    <p>{{page.intro|richtext}}</p>
```

Comment:
Won't work as expected with a `<p>` parent ... Since Wagtail Richtext has a paragraph element as a root, it always ignores other paragraphs you attempt to put it in (`p` cannot be a child of `p`). Changed to a `div` if you need it, or do without it.